Source/Fuse.Nodes/Visual.ZOrder.uno

```diff
@@ -6,219 +6,123 @@ namespace Fuse
 {
 	public partial class Visual
 	{
-		public abstract void Draw(DrawContext dc);
-
-		public bool HasVisualChildren { get { return _zOrder != null && _zOrder.Count > 0; } }
-
-		[Obsolete("Use FirstChild<Visual>() instead")]
-		public Visual FirstVisualChild
-		{ 
-			get
-			{
-				return FirstChild<Visual>();
-			}
-		}
-
-		/** Returns the visual child with the given index. 
-		
-			For performance reasons, avoid using this function. 
+		float _zOffset = 0;
+		/**
+			Specifics a ZOffset, higher values are in front of other nodes. Only used by certain Node's,
+			such as `Panel`. The ZLayer has priority, then ZOffset, then ZOffsetNatural.
```

Comment:
I think this comment is now wrong. I belive you've elevated ZOrder handling completely to the Visual level -- it's not longer handled by the Layout system. THat might warrant a CHANGELOG entry, though I'm not sure it would affect any user (since only LayoutControl really used multiple children before).  THe comment should be correct in any case.